### ISSUE TITLE
[IBM Power Up] Your Project Title

### ISSUE THREAD

**Issue by Yiningxie** [XanaduAI/QHack2022#51]
### Team Name: QH

*Your team's name (matching the name used on the QHack Coding Challenges, if applicable)*

### Project Description: Analyzing Interaction between Proteome and Genome with Quantum Computers

*A brief description of your project (1-2 paragraphs).*

Severe acute respiratory syndrome coronavirus 2 (SARS CoV 2) for COVID-19, including its variants, has been wide spread globally. It affects all ages, races and various medical conditions. Using traditional computers is quite challenging to analyze interactions between proteome and genome, including gene-gene Interactions among virus and human genetics. This project is to use currently available quantum computers to analyze interactions between proteome and genome, including gene-gene interactions in SARS CoV 2 and human genetics.

Virology and genetics have been two fields that I’m very interested since my medical school. After my honorable graduation from Tongji Medical School at Huazhong University of Science and Technology where I learned Western Medicine and some Chinese Medicine, I did my Residency on Internal Medicine at Union (Xie He) Hospital. I then went to Germany and did my doctoral thesis at Munich University. After obtaining my doctoral degree with honor from Munich University, I came to University of California at Los Angeles (UCLA) for my post-doctoral fellowship and subsequently worked there. I also took and successfully passed the U.S. National Board Step-1, National Board Step-2 and Clinical Skill Assessment (CSA), and am certified by the U.S. Educational Commission for Foreign Medical Graduates (ECFMG).

In 2006, I moved from Los Angeles (UCLA) to Washington, DC for my job at the National Institutes at Health (NIH), and further strength my interests in Virology and genetics, particular at the time of current COVID-19 pandemics. My e-mail is [yiningxie3@gmail.com](mailto:yiningxie3@gmail.com) and my phone # is 240-453-1534. I am looking forward to hearing from you about my draft of this QHack Open Hackathon project.

Many thanks, Sincerely,

Yining Xie

### Source code: 
The Quantum Programming Language Twist from MIT will be used in conjunction with PenneLane (Pythone, Numby, PyTorch, TensorFlow, JAX interface, Cirq, Strawberrybields, Qiskitt, Forest, etc).

*A hyperlink to the draft source code for your team's hackathon project (e.g., a GitHub repo).*

https://doi.org/10.1145/3498691
Two Attachments here: 3498691.pdf; appendices.pdf
[3498691.pdf](https://github.com/XanaduAI/QHack/files/8121894/3498691.pdf)
[appendices.pdf](https://github.com/XanaduAI/QHack/files/8121896/appendices.pdf)

### Resource Estimate: 

If awarded, the access to IBM Quantum machine with IBM 16-qubit QPU will be used to finish this project.

The Quantum Programming Language Twist from MIT will be used in conjunction with PenneLane (Pythone, Numby, PyTorch, TensorFlow, JAX interface, Cirq, Strawberrybields, Qiskitt, Forest, etc) as stated the above.

*A 1-2 paragraph written Resource Estimate, indicating how you expect to use the IBM 16-qubit QPU, if awarded, to finish your Open Hackathon project.*

**Comment by isaacdevlugt** [comment]
Thank you for your Power Up submission! As a reminder, the final deadline for your project is February 25 at 17h00 EST. Submissions should be done here: https://github.com/XanaduAI/QHack/issues/new?assignees=&labels=&template=open_hackathon.md&title=%5BENTRY%5D+Your+Project+Title

This issue will be closed shortly.

Good luck!
